print_env.sh

```diff
@@ -0,0 +1,34 @@
+echo "***git log --decorate -n 1***"
+git log --decorate -n 1
+echo " "
+
+echo "***cat /etc/*-release***"
+cat /etc/*-release
+echo " "
+
+echo "***uname -a***"
+uname -a
+echo " "
+
+echo "***nvidia-smi***"
+nvidia-smi
+echo " "
+
+echo '***conda list "pandas|arrow|numpy|^python$"***'
+conda list 'pandas|arrow|numpy|^python$'
```

Comment:
It would be good to detect if a user has conda and fallback to pip otherwise. Not needed for v1 though.

Reply:
I agree that's a good idea, but that's probably beyond my bash-fu skills. Would love it if someone could add in the future.

Reply:
@raydouglass when you have a moment can you help here?

Reply:
Something like this:
`which conda && conda list 'pandas|arrow|numpy|^python$' || { pip list | grep -E "pandas|arrow|numpy"; python --version; }`

Reply:
Something like this: `which conda && conda list 'pandas|arrow|numpy|^python$' || { pip list | grep -E "pandas|arrow|numpy"; python --version; }`